modules/adding-users-to-specialized-data-science-user-groups.adoc

```diff
@@ -0,0 +1,53 @@
+:_module-type: PROCEDURE
+
+[id='adding-users-to-specialized-openshift-data-science-user-groups_{context}']
```

Comment:
This might be an ID we need to change to remove the product name. Otherwise, OpenShift Data Science will appear in the upstream URL.

Reply:
yes - thanks for catching